scripts/configure_dev.sh

```diff
@@ -69,13 +69,24 @@ function install_windows_shellcheck() {
 }
 
 if [ "${OS}" = "linux" ]; then
+    DISTRIB=$("$SCRIPTPATH"/distribtype.sh)
```

Comment:
if this is the *only* place where you need to call `os-release`, than it would make sense to have it called explicitly here instead of creating a separate file.

Reply:
Ok, I am removing this file.